vnext/Microsoft.ReactNative.Cxx/CppWinRTIncludes.h

```diff
@@ -17,21 +17,21 @@ namespace xaml = winrt::Windows::UI::Xaml;
 namespace comp = winrt::Windows::UI::Composition;
 namespace ui = winrt::Windows::UI;
 namespace winrt {
-namespace system = winrt::Windows::System;
+namespace dispatching = winrt::Windows::System;
 using ColorHelper = winrt::Windows::UI::ColorHelper;
 using Colors = winrt::Windows::UI::Colors;
 } // namespace winrt
 #else
 
-#include <winrt/Microsoft.System.h>
+#include <winrt/Microsoft.UI.Dispatching.h>
```

Comment:
Perhaps this include causes the issue?

Seems like this header is exclusive to WinUI3/WinAppSDK.
See https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.dispatching?view=winui-3.0

Reply:
this change is due to some apis moving between namespaces across versions of winappsdk

Reply:
yep, what Alex said. This is mostly just a rename and will only be imported when --useWinui3 is specified.